package.json

```
@@ -6,7 +6,10 @@
     "format": "prettier --write .",
     "check-format": "prettier --check .",
     "lint": "eslint . && ejslint ./template/src",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test": "tests/test.sh",
```

I'd prefer to use the `concurrently` npm package for this, which will let us delete the test.sh file and works cross platform on windows. The advantage of running them concurrently is that they'll be faster.

https://www.npmjs.com/package/concurrently

```suggestion
    "test": "concurrently -n pnpm:test:*"
```

If running them concurrently isn't possible for any reason, then we can just use `&&` syntax in the package.json:

```sh
    "test": "pnpm run test:vue2 && pnpm run test:vue3"
```

Regarding setting the current working directory for the tests, I suggest having testVue.sh assume that the cwd is the root of vue-cli-plugin-single-spa, and set the `i` variable to be `$PWD/test`

---

Jup concurrently is also way faster and helps visibility!
Done in 0ce06e7